### DISCUSSION TITLE
Need help deploying codes for POS

### DISCUSSION THREAD

**Discussion by mharisiqbal** [OCA/pos#1271] [General]
Hi guys. I am new on Odoo, I run a small supermarket and was wondering if I could deploy these codes to build up my POS. However, I dont know where to begin. Could somebody please guide me on how to deploy these codes? Thanks in advance

**Comment by legalsylvain** [comment]
Hi. this is not the place to ask help for general usage of odoo. (how to install modules, etc...) 
You could hire a IT people to do it, or ask for an integrator in your contry.

kind regards.
